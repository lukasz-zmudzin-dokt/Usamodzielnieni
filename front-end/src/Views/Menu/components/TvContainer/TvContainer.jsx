import React from "react";
import { VideoField } from "components";

const TvContainer = () => {
  return (
    <div className="tvContainer">
      <div className="tvContainer__shapes">
        <div className="tvContainer__shapes__left">
          <img alt="" src="/left_tv.svg" />
        </div>
        <div className="tvContainer__shapes__right">
          <img alt="" src="/right_tv.svg" />
        </div>
      </div>
      <div className="tvContainer__tv">
        <img alt="" src="/tv.png" />
        <div className="tvContainer__tv__video__container">
          <div className="tvContainer__tv__video">
            <div className="tvContainer__tv__video__space">
              <VideoField id="1" />
            </div>
          </div>
        </div>
      </div>
    </div>
  );
};

export default TvContainer;
