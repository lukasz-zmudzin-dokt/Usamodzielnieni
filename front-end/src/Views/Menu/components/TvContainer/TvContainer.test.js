import React from "react";
import { render } from "@testing-library/react";
import TvContainer from "./TvContainer";

describe("TvContainer", () => {
  it("should render without crashing", () => {
    const { container } = render(<TvContainer />);
    expect(container).toMatchSnapshot();
  });
});
