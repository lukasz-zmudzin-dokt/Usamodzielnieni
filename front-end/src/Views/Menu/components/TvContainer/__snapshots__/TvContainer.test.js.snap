// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TvContainer should render without crashing 1`] = `
<div>
  <div
    class="tvContainer"
  >
    <div
      class="tvContainer__shapes"
    >
      <div
        class="tvContainer__shapes__left"
      >
        <img
          alt=""
          src="/left_tv.svg"
        />
      </div>
      <div
        class="tvContainer__shapes__right"
      >
        <img
          alt=""
          src="/right_tv.svg"
        />
      </div>
    </div>
    <div
      class="tvContainer__tv"
    >
      <img
        alt=""
        src="/tv.png"
      />
      <div
        class="tvContainer__tv__video__container"
      >
        <div
          class="tvContainer__tv__video"
        >
          <div
            class="tvContainer__tv__video__space"
          >
            <div>
              VideoField
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
